### DISCUSSION TITLE
Is there a way to edit asset files without rebuilding the project?

### DISCUSSION THREAD

**Discussion by millenniumbrain** [hyperion-project/hyperion.ng#1309] [Q&A] [answered]
Is there a way to edit js files and see the changes in the browser without rebuilding the project? I'm playing around with it in Ubuntu and the JavaScript files only change in browser when a do a rebuild. Trying to play around with the JavaScript to see if I can contribute to this project.

**Comment by Paulchen-Panther** [comment, marked as answer]
Yes, there is. In Hyperion WebUI set the Settings level to "Expert" and under the entry "System" -> "Web configuration" set the "Document Root" to the "asset/webconfig" directory. After that changes to js/html files should be displayed.

![image](https://user-images.githubusercontent.com/16664240/129485277-8a4c632c-6b6c-48b2-8667-2db272b309e3.png)
![image](https://user-images.githubusercontent.com/16664240/129485297-b6bcb51e-06f7-479e-8cf3-b2ed44b4832d.png)
![image](https://user-images.githubusercontent.com/16664240/129485307-e33ca114-5cdb-47f6-9ba0-dbb933d66bcd.png)
![image](https://user-images.githubusercontent.com/16664240/129485344-db9fd17b-815e-4713-a941-13dcdc9c8e5e.png)
